datasets/parsinlu_reading_comprehension/parsinlu_reading_comprehension.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ParsiNLU Persian reading comprehension task"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import datasets
+
+
+logger = datasets.logging.get_logger(__name__)
+
+_CITATION = """\
+@article{huggingface:dataset,
+    title = {ParsiNLU: A Suite of Language Understanding Challenges for Persian},
+    authors = {Khashabi, Daniel and Cohan, Arman and Shakeri, Siamak and Hosseini, Pedram and Pezeshkpour, Pouya and Alikhani, Malihe and Aminnaseri, Moin and Bitaab, Marzieh and Brahman, Faeze and Ghazarian, Sarik and others},
+    year={2020}
+    journal = {arXiv e-prints},
+    eprint = {2012.06154},    
+}
+"""
+
+# You can copy an official description
+_DESCRIPTION = """\
+A Persian reading comprehenion task (generating an answer, given a question and a context paragraph). 
+The questions are mined using Google auto-complete, their answers and the corresponding evidence documents are manually annotated by native speakers.    
+"""
+
+_HOMEPAGE = "https://github.com/persiannlp/parsinlu/"
+
+_LICENSE = "CC BY-NC-SA 4.0"
+
+_URL = "https://raw.githubusercontent.com/persiannlp/parsinlu/master/data/reading_comprehension/"
+_URLs = {
+    "train": _URL + "train.jsonl",
+    "dev": _URL + "dev.jsonl",
+    "test": _URL + "eval.jsonl",
+}
+
+
+class ParsinluReadingComprehension(datasets.GeneratorBasedBuilder):
+    """ParsiNLU Persian reading comprehension task."""
+
+    VERSION = datasets.Version("1.0.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="parsinlu-repo", version=VERSION, description="ParsiNLU repository: reading-comprehension"
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = (
+        "parsinlu-repo"  # It's not mandatory to have a default configuration. Just use one if it make sense.
+    )
+
+    def _info(self):
+        features = datasets.Features(
+            {
+                "question": datasets.Value("string"),
+                "url": datasets.Value("string"),
+                "passage": datasets.Value("string"),
```

Comment:
Could you rename this field "context" for consistency with the other squad-like datasets ?

You'll need to regenerate the dataset_infos.json file after this change

Reply:
Done.